packages/cli/utils/version.ts

```diff
@@ -0,0 +1,64 @@
+import { readFile } from "fs/promises";
+import { join } from "path";
+import { gt } from "semver";
+
+import { MODULE_ROOT } from "./constants.js";
+
+export async function current() {
```

Comment:
This is what we do in the node-sdk to get the version from package.json: https://github.com/bucketco/bucket-javascript-sdk/blob/main/packages/node-sdk/src/config.ts#L3

Can we use that here too?

Reply:
Not possible, as we're using module and that's not supported. There is another way but there's a nasty warning about it being experimental so I'd rather not use that if this method works.